cinema/serializer.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True, max_length=255)
+    description = serializers.CharField(required=True, max_length=255)
+    duration = serializers.IntegerField(required=True)
+
+    def create(self, validated_data: dict) -> Movie:
+        return Movie.objects.create(**validated_data)
+
+    def update(self, instance: Movie, validated_data:dict) -> Movie:
```

Comment:
Format your code appropriately
```suggestion
    def update(self, instance: Movie, validated_data: dict) -> Movie:
```